### PR TITLE
fix duplicate edgeinfo

### PR DESCRIPTION
**Pull request by kevinkreiser**
it turns out comparing references makes no sense... stupid muscle memory

added a test to prove it was working as it should

also added the ability to load more than one pbf at once

### CODE REVIEW

File: valhalla/mjolnir/graphtilebuilder.h [hunk 1]
```diff
@@ -24,8 +24,7 @@
 namespace valhalla {
 namespace mjolnir {
 
-using edge_tuple = std::tuple<const uint32_t,
-    const valhalla::baldr::GraphId&, const valhalla::baldr::GraphId&>;
```
Comment (kevinkreiser) [inline valhalla/mjolnir/graphtilebuilder.h]: this idiocy was the culprit. apologies all

File: test/graphtilebuilder.cc [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  //add edge info for node 1 to node 0
+  test.AddEdgeInfo(0, GraphId(0,2,1), GraphId(0,2,0), {{1, 1}, {0, 0}}, {"einzelweg"});
+  if(test.edge_offset_map.size() != 1)
+    throw std::runtime_error("There should still be exactly one of these in here");
+}
+
+}
+
+int main() {
+  test::suite suite("edgeinfobuilder");
```
Comment (dgearhart) [inline test/graphtilebuilder.cc]: graph tile buiilder?

**Comment by dgearhart** [comment]
otherwise :+1:

**Comment by dnesbitt61** [comment]
:+1:

**Pull request merged**